site/pages/api/delete-post.api.ts

```diff
@@ -0,0 +1,69 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { DISRUPTION_DETAIL_PAGE_PATH, REVIEW_DISRUPTION_PAGE_PATH } from "../../constants";
+import { removeSocialMediaPostFromDisruption, upsertSocialMediaPost } from "../../data/dynamo";
+import { SocialMediaPostTransformed as SocialMediaPost } from "../../schemas/social-media.schema";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
+import { getSession } from "../../utils/apiUtils/auth";
+
+const deleteConsequence = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
```

Comment:
Update the variable name to remove consequence

Reply:
done changed to post